recipes/pyjion/meta.yaml

```
@@ -0,0 +1,80 @@
+{% set name = "pyjion" %}
+{% set version = "1.2.6" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/tonybaloney/Pyjion.git
+  # https://github.com/tonybaloney/Pyjion/tree/1.2.6
+  git_rev: 66a772d48825e2511b9c6e8bbcc739d84f84e800
```

https://github.com/conda-forge/staged-recipes/pull/17937#discussion_r801656393
```suggestion
  git_rev: {{ version }}
```